conf/workloads/micro/dfsioe.conf

```diff
@@ -0,0 +1,46 @@
+#DFSIOE
+#Unit: number_of_file -> as name suggests
+#Unit: filesize -> MB of file size
+hibench.dfsioe.tiny.read.number_of_files        16
+hibench.dfsioe.tiny.read.file_size              1
+hibench.dfsioe.tiny.write.number_of_files       16
+hibench.dfsioe.tiny.write.file_size             1
+
+hibench.dfsioe.small.read.number_of_files       32
+hibench.dfsioe.small.read.file_size             10
+hibench.dfsioe.small.write.number_of_files      32
+hibench.dfsioe.small.write.file_size            10
+
+hibench.dfsioe.large.read.number_of_files       64
+hibench.dfsioe.large.read.file_size             10
+hibench.dfsioe.large.write.number_of_files      64
+hibench.dfsioe.large.write.file_size            10
+
+hibench.dfsioe.huge.read.number_of_files        256
+hibench.dfsioe.huge.read.file_size              100
+hibench.dfsioe.huge.write.number_of_files       256
+hibench.dfsioe.huge.write.file_size             100
+
+hibench.dfsioe.gigantic.read.number_of_files    512
+hibench.dfsioe.gigantic.read.file_size          400
+hibench.dfsioe.gigantic.write.number_of_files   512
+hibench.dfsioe.gigantic.write.file_size         400
+
+hibench.dfsioe.bigdata.read.number_of_files     2048
+hibench.dfsioe.bigdata.read.file_size           1000
+hibench.dfsioe.bigdata.write.number_of_files    2048
+hibench.dfsioe.bigdata.write.file_size          1000
+
+
+hibench.sparkbench.jar      ${hibench.home}/sparkbench/micro/target/sparkbench-micro-6.0-SNAPSHOT-${hibench.spark.version}-jar-with-dependencies.jar
+
+hibench.dfsioe.read.number_of_files     ${hibench.dfsioe.${hibench.scale.profile}.read.number_of_files}
+hibench.dfsioe.read.file_size           ${hibench.dfsioe.${hibench.scale.profile}.read.file_size}
+hibench.dfsioe.write.number_of_files    ${hibench.dfsioe.${hibench.scale.profile}.write.number_of_files}
+hibench.dfsioe.write.file_size          ${hibench.dfsioe.${hibench.scale.profile}.write.file_size}
+
+# export for shell script
+hibench.dfsioe.base.hdfs                ${hibench.hdfs.data.dir}/Dfsioe
+hibench.workload.input                  ${hibench.dfsioe.base.hdfs}/Input
+hibench.workload.output                 ${hibench.dfsioe.base.hdfs}/Output
+
```

Comment:
Can you remove `hibench.dfsioe.base.hdfs` and change the above three lines to two:
hibench.workload.input                  ${hibench.hdfs.data.dir}/Dfsioe/Input
hibench.workload.output               ${hibench.hdfs.data.dir}/Dfsioe/Output